### ISSUE TITLE
Not able to join a group

### ISSUE THREAD

**Issue by Lukman-Abdulrahman** [signalapp/Signal-Android#10389]
- [ x] I have searched open and closed issues for duplicates
- [x ] I am submitting a bug report for existing functionality that does not work as intended
- [x ] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x ] This isn't a feature request or a discussion topic

----------------------------------------

### Bug Description 
I am not able to join a group. Even with a good internet, when I get invite to join a group and I clicked it, it takes me to the app but the loading screen seems to load forever. Even when am added, I don't see messages on the group. 
And also I can't create a group. It seems group doesn't work on the app. 
### Steps to reproduce
- Use the signal app on Android  
- Get a group invitation. 
- Click it and go to the Signal App. 

**Actual result:** 
The loading screen loads forever without adding me to the intended group. And also messages sent in the group I was added to won't even show up that I was added. 
**Expected result:**
I should be able to do all this successfully 

### Screenshots
![Screenshot_20210109-062019](https://user-images.githubusercontent.com/73845970/104084047-fd3f4280-5243-11eb-97aa-13690b876191.png)

### Device info

**Device:**ZTE BLADE V7 MAX
**Android version:** 6.0
**Signal version:** 5.0.8

### Link to debug log

https://debuglogs.org/a3881a50aa14ffa99cffcb8504d90949822494b842d798523f7096a313dadf54

**Comment by androdebugur** [comment]
Hi,
Quickly looking at the error log I found this,

`W PhoneNumberFormatter: Error type: NOT_A_NUMBER. The string supplied did not seem to be a phone number.`

Is it possible for you to share the link that you have received to join a group, of-course you can redact some information.
Or to compare on your own by creating another group link.

**Comment by Lukman-Abdulrahman** [comment]
I am also not able to create a group. And this is the invite group link. 
https://signal.group/#CjQKIIpCR-vQzbRZNcfyJPb_zMtJY4Al_HSumV-Hh1Le4kDYEhAWTOXKytP0ZLVb3m9migDs

**Comment by androdebugur** [comment]
> I am also not able to create a group. And this is the invite group link.
> https://signal.group/#CjQKIIpCR-vQzbRZNcfyJPb_zMtJY4Al_HSumV-Hh1Le4kDYEhAWTOXKytP0ZLVb3m9migDs

Thank you for sharing a link to join a group.

**Comment by androdebugur** [comment]
Further looking through the log, looks like internet connection issue. In some countries Signal might be blocked.

`java.io.IOException: No connection!`

and 

`failed to connect to storage.signal.org/216.239.32.21 (port 443) after 30000ms`

Can you verify that internet connection is not a issue by, opening this link in your browser [http://storage.signal.org/](http://storage.signal.org/) or [https://storage.signal.org/](https://storage.signal.org/).

You should see something like this if internet connection is not the issue here.
`{"code":404,"message":"HTTP 404 Not Found"}`

**Comment by Lukman-Abdulrahman** [comment]
Yes I am getting that exact code.

**Comment by Lukman-Abdulrahman** [comment]
I just changed my network sim and I was able to finally close it. 
So for anyone having similar issue. I will suggest you change yours too. 
And before I close this, I want to suggest some features. 
-Adding stories like the whatsapp status 
-Ability to add stickers made by the users themselves. 
Those are the two I remember for now. 
Thanks everyone for answering me, I wish I have the skill to contribute. 


Thanks.
